src/main/webapp/WEB-INF/view/profilepage.jsp

```diff
@@ -0,0 +1,48 @@
+<%@ page import="codeu.model.data.User" %>
+
+<!DOCTYPE html>
+<html>
+<head>
+ <title>Profile Page</title>
+ <link rel="stylesheet" href="/css/main.css">
+ <link rel="stylesheet" href="/css/profilepage.css">
+ <style>
+   label {
+     display: inline-block;
+     width: 100px;
+   }
+ </style>
+</head>
+<body>
+    <nav>
```

Comment:
Let's wait for Cait to merge the nav bar change, then add the profile link there.